src/api/om-api-LW/graphics.lisp

```diff
@@ -347,6 +348,10 @@
      :font (gp::find-best-font view font)
      )))
 
+;; this is more general than "graphics" ... move to tools.lisp?

```

Comment:
... it's alo used for string rendering, somehow, so I guess we can keep it here

Reply:
Btw note that there exist a function `string-to-list` in the `om` package (string.lisp) which does pretty much the same. The one from LW is most likely more efficient, but not necessarily!. The one from OM has only one delimiter, but could be easily extended to support a list of delimiters as well.

Reply:
good idea, I'm for using what's there already. Looks like we could use a `&rest` in the arguments for `string-until-char`, that maintain compatibility with the other calls to that function. Then do similarly for string-to-list. Would have to accept actual characters in addition to (or instead of) single-character strings, so that we can use #\Return

Reply:
or make it a method and specialize for different argument types?

Reply:
All possible options. I think all the callers of this function use it with single-character string anyway, and teh function si kind of meabt for that, since it does `(subseq string (+ index 1))` ...

Reply:
Actually I don't need split-sequence, or to change the existing functions, since there is also `string-lines-to-list`. See new commit 922c6d0. I could get rid of om-split-sequence now, with a rebase and force push